src/routes/Plugin/Common/index.js

```diff
@@ -624,7 +691,27 @@ export default class Common extends Component {
     ];
 
     return (
-      <div className="plug-content-wrap">
+      <div className="plug-content-wrap"> 
+        <Row style={{ marginBottom: '5px', display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
+          <div style={{ display: 'flex',alignItems:'end',flex:1,margin:0 }}>
+            <Title level={2} style={{ textTransform: 'capitalize', margin: '0 20px 0 0' }}>
+              {name}
+            </Title>
+            <Title level={3} type="secondary" style={{margin:0}}>{role}</Title>
+          </div>
+          <div style={{ display: 'flex',alignItems:'end',gap:10 }}> 
+            <Switch
```

Comment:
Hi Maybe you should add events that you can modify.
<img width="1208" alt="image" src="https://user-images.githubusercontent.com/33576070/230708486-5546514c-efa2-4974-9935-7c49fd400605.png">

Reply:
Thanks for the tip.
I can attempt on it, can I get some more idea on it, I didn't quite get  an understanding on what you mentioned

Reply:
<img width="1417" alt="image" src="https://user-images.githubusercontent.com/33576070/230725640-fa75eb84-5f79-4205-8d26-4963c08a4e99.png">

The switch button is used to quickly enable and disable plug-ins.
You can do this using the editing interface.[plugin/update]

Reply:
Hi, currently the switch can't be toggled ,it can only be changed by clicking edit and changing the status in the modal.
Should I allow changing the status by clicking the switch?

Reply:
yes，
You can also replace them with tags so you don't have to add modification events

<img width="74" alt="image" src="https://user-images.githubusercontent.com/33576070/230729256-598b9d42-ef08-4bfc-9f47-4872fa3529ab.png">